HHS/Head-Start-TTADP
# fix for event import

## Description of change

This is a small change to leave e-mail addresses as is for import.

Also fix an issue with the order of the migrations being run.


## How to test

All tests pass.

Make sure you can visit the event landing without a 500 (run migrations first).

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
